src/patternfly/components/CodeEditor/examples/CodeEditor.md

```diff
@@ -104,7 +104,7 @@ cssPrefix: pf-v5-c-code-editor
     {{#> code-editor-keyboard-shortcuts}}
       {{#> button button--modifier="pf-m-link"}}
         {{#> button-icon button-icon--modifier="pf-m-start"}}
-          <i class="pf-v5-c-icon pf-v5-c-icon-help" aria-hidden="true"></i>
+          <i class="pf-v5-pficon pf-v5-pficon-help" aria-hidden="true"></i>
```

Comment:
`pf-v5-pficon` only appears to be a class on the progress stepper but it's on multiple examples including this one. Is it needed?

Reply:
I noticed that, too. It used to serve a similar role as `.fa/fas` for a FA icon - it attached base styles for all icons. But at some point those styles were moved to each icon selector, so it isn't technically required anymore. Though in at least one component, we are styling `.pficon` (or `.pf-v5-icon` now) for when any of these icons are used, so I can see utility in keeping it. If so, maybe we should create a follow up to add it to any examples that are missing the class. All the spots I saw that were missing it in this PR were already missing it, so not a result of this PR.

Reply:
I agree, not because of this, but it would be good to clean this up and be consistent, i.e. add it always or add it if it's needed for styling but right now it's kind of random. I'll create an issue to track a decision on what and then clean them up.

Reply:
Added https://github.com/patternfly/patternfly/issues/5552